apps/core/src/auth/cognito-jwt-auth.guard.ts

```diff
@@ -52,8 +57,15 @@ export class CognitoJwtAuthGuard implements CanActivate {
     );
 
     try {
-      await this.config.cognitoJwtVerifier.verify(this.getBearerToken(context));
-      this.logger.log(`Request authorized to ${controllerName}.${handlerName}`);
+      const httpRequest = context.switchToHttp().getRequest<HttpRequest>(),
+        { url } = httpRequest;
+
+      const { sub } = await this.config.cognitoJwtVerifier.verify(
+        this.getBearerToken(httpRequest),
+      );
+      this.logger.log(
+        `User ID "${sub}" authorized to ${controllerName}.${handlerName} (url: "${url}")`,
```

Comment:
Do we know if logging this token is okay with security?

Reply:
The `sub` is an UUID of the authenticated user, not any sensitive information (no way to carry out attack with `sub`) 👍

https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-the-id-token.html
> sub
A unique identifier (UUID), or subject, for the authenticated user. The username might not be unique in your user pool. The sub claim is the best way to identify a given user.